scenario-definitions.yaml

```
@@ -0,0 +1,29 @@
+---
+products:
+  opensuse-*-dev-x86_64:
+    distri: opensuse
+    flavor: dev
+    arch: x86_64
+    version: Tumbleweed
+
+machines:
+  64bit:
+    backend: qemu
+    settings:
+      HDDSIZEGB: "20"
+      QEMUCPU: host
+      VIRTIO_CONSOLE: "1"
```

It's better to disable the network for now explicitly
```suggestion
      VIRTIO_CONSOLE: "1"
      OFFLINE_SUT: "1"
```